modules/document_repository/ajax/documentEditUpload.php

```diff
@@ -89,26 +86,21 @@
                  'version'       => $version,
                  'File_name'     => $fileName,
                  'File_size'     => $fileSize,
-                 'Data_dir'      => $fileBase,
-                 'uploaded_by'   => $puser,
+                 'Data_dir'      => $fullPath,
```

Comment:
does changing this from relative to full path affect anywhere else where the `document_repository` table is used?